### PR TITLE
Add the BiggerPockets site

### PR DESCRIPTION
**Pull request by jonade**

### CODE REVIEW

**Review by stephengroat — DISMISSED** [review]

**Comment by dsoegijono** [comment]
Would [this link](https://www.biggerpockets.com/forums/25/topics/232201-new-biggerpockets-security-improvements) be useful as a doc?

**Comment by jonade** [comment]
> Would [this link](https://www.biggerpockets.com/forums/25/topics/232201-new-biggerpockets-security-improvements) be useful as a doc?

Possibly. I didn't include this link because it's actually a forum thread, rather than an article/document.

File: _data/social.yml [hunk 1]
```diff
@@ -26,6 +26,12 @@ websites:
       tfa: No
       twitter: Badoo
 
+    - name: BiggerPockets
+      url: https://www.biggerpockets.com
+      img: biggerpockets.png
+      tfa: Yes
+      sms: Yes
+
```
Comment (kmpoppe) [inline _data/social.yml]:
````suggestion:
     doc: https://www.biggerpockets.com/forums/25/topics/232201

````

Comment (kmpoppe) [inline _data/social.yml]: Adding documentation link as follows

Comment (kmpoppe) [reply _data/social.yml]:
```suggestion
     doc: https://www.biggerpockets.com/forums/25/topics/232201
     
```

**Review by kmpoppe — APPROVED** [review]

**Review by jtagcat — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**